pom.xml

```diff
@@ -46,6 +46,7 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-web</artifactId>
 			<version>${spring.version}</version>
+			<scope>provided</scope>
```

Comment:
This needs to be excluded in the alfresco connector and not marked as provided here. This may break other users of this project.